changed.d

```diff
@@ -101,8 +101,10 @@ auto getIssues(string revRange)
     foreach (repo; ["dmd", "druntime", "phobos", "dlang.org", "tools", "installer"]
              .map!(r => buildPath("..", r)))
     {
-        auto cmd = ["git", "-C", repo, "fetch", "upstream", "--tags"];
-        auto p = pipeProcess(cmd, Redirect.stdout);
+        // ensure that changed_upstream exists (we ignore failures)
+        auto cmd = ["git", "-C", repo, "fetch", "--tags", "https://github.com/dlang/" ~ repo.baseName,
+                           "+refs/heads/*:refs/remotes/upstream/*"];
+        auto p = pipeProcess(cmd, Redirect.stdout, ["GIT_NAMESPACE": "changed"]);
         enforce(wait(p.pid) == 0, "Failed to execute '%(%s %)'.".format(cmd));
 
         cmd = ["git", "-C", repo, "log", revRange];
```

Comment:
This needs to have the `GIT_NAMESPACE` thing too, otherwise it won't use the correct refs. Right?